src/trackers/helpers/rule.js

```diff
@@ -0,0 +1,54 @@
+const sharedData = require('./sharedData.js')
+const tldts = require('tldts')
+
+// Adds example sites for each rule if enabled in the config file
+// Half of the sites can be chosen from a top sites list provided in the config. 
+// The other half are randomly chosen form the list of sites a request was found on
+function getExampleSites (sites, limit) {
+    // remove IPs from example site list, if any
+    sites = sites.filter(site => {
+        if (!site) {
+            return false
+        }
+        const parsedSite = tldts.parse(site)
+        return !parsedSite.isIp
+    })
+
+    if (sites.length <= limit) {
+        return sites
+    }
+
+    const idxList = []
+
+    // half of the list is topExampleSites if given
+    if (sharedData.topExampleSitesSet) {
+        const intersection = sites.filter(x => sharedData.topExampleSitesSet.has(x))
+        intersection.slice(0, Math.floor(limit/2)).map(domain => idxList.push(sites.indexOf(domain)))
```

Comment:
```suggestion
        sites
            .filter(x => sharedData.topExampleSitesSet.has(x))
            .slice(0, Math.floor(limit/2))
            .forEach(domain => idxList.push(sites.indexOf(domain)))
```

nit: removed variable as we don't need it, map->forEach, formatting